svelte-vite-ts/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "svelte-vite",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
+    "check": "svelte-check --tsconfig ./tsconfig.json"
+  },
+  "devDependencies": {
+    "@sveltejs/vite-plugin-svelte": "^1.0.2",
+    "@tsconfig/svelte": "^2.0.1",
+    "autoprefixer": "^10.4.4",
+    "cypress": "10.7.0",
```

Comment:
Expected release, to test locally replace with a binary like `https://cdn.cypress.io/beta/npm/10.7.0/darwin-x64/develop-f6eaad40e1836fa9db87c60defa5ae6f390c8fd8/cypress.tgz`, more can be found https://app.circleci.com/pipelines/github/cypress-io/cypress?branch=develop